### PR TITLE
snoop: add page

### PR DESCRIPTION
**Pull request by HaxxonHax**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/sunos/snoop.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# snoop
+
+> Network packeet sniffer.
```
Comment (schneiderl) [inline pages/sunos/snoop.md]:
```suggestion
> Network packet sniffer.
```

File: pages/sunos/snoop.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> SunOS equivalent of tcpdump.
+
+- Capture packets on network interface e1000g0:
+
+`snoop -d e1000g0`
```
Comment (schneiderl) [inline pages/sunos/snoop.md]:
```suggestion
`snoop -d {e1000g0}`
```

File: pages/sunos/snoop.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Capture packets to a file:
+
+`snoop -o {{filename}}`
+
+- Display verbose protocol layer summary of packets from a file:
+
+`snoop -V -i {{filename}}`
+
+- Capture network packets that come from hostname and go to port 8000:
```
Comment (schneiderl) [inline pages/sunos/snoop.md]:
```suggestion
- Capture network packets that come from a hostname and go to a given port:
```

File: pages/sunos/snoop.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`snoop to port 8000 from host {{hostname}}`
```
Comment (schneiderl) [inline pages/sunos/snoop.md]:
```suggestion
`snoop to port {port} from host {{hostname}}`
```

File: pages/sunos/snoop.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# snoop
+
+> Network packeet sniffer.
+> SunOS equivalent of tcpdump.
+
+- Capture packets on network interface e1000g0:
```
Comment (mebeim) [inline pages/sunos/snoop.md]:
```suggestion
- Capture packets on a specific network interface:
```
I'd reword this to make it cleaner.

File: pages/sunos/snoop.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`snoop -d e1000g0`
+
+- Capture packets to a file:
```
Comment (mebeim) [inline pages/sunos/snoop.md]:
```suggestion
- Capture packets and save them to a file:
```
Better wording.

File: pages/sunos/snoop.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display verbose protocol layer summary of packets from a file:
+
+`snoop -V -i {{filename}}`
+
+- Capture network packets that come from hostname and go to port 8000:
+
+`snoop to port 8000 from host {{hostname}}`
+
+- Capture and show network packets in hex and ASCII between two IP addresses:
```
Comment (mebeim) [inline pages/sunos/snoop.md]:
```suggestion
- Capture and show an hex-dump of network packets exchanged between two IP addresses:
```
Clearer.

File: pages/sunos/snoop.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`snoop -x0 -p4 {{ip address 1}} {{ip address 2}}`
```
Comment (mebeim) [inline pages/sunos/snoop.md]:
```suggestion
`snoop -x0 -p4 {{ip_address_1}} {{ip_address_2}}`
```
Use underscores instead of spaces.

File: pages/sunos/snoop.md (hunk 2 above)
Comment (HaxxonHax) [reply pages/sunos/snoop.md]: Thanks!

File: pages/sunos/snoop.md (hunk 5 above)
Comment (HaxxonHax) [reply pages/sunos/snoop.md]: Thanks!

File: pages/sunos/snoop.md (hunk 6 above)
Comment (HaxxonHax) [reply pages/sunos/snoop.md]: I'm not sure if this is better wording.  The use of `and` implies that the capture is both sent to the screen **and** sent to a file.

File: pages/sunos/snoop.md (hunk 7 above)
Comment (HaxxonHax) [reply pages/sunos/snoop.md]: British pronunciation would use `an hex-dump`, while American English would use `a hex-dump`.

File: pages/sunos/snoop.md (hunk 4 above)
Comment (HaxxonHax) [reply pages/sunos/snoop.md]: Thanks!

File: pages/sunos/snoop.md (hunk 3 above)
Comment (HaxxonHax) [reply pages/sunos/snoop.md]: Thanks!

File: pages/sunos/snoop.md (hunk 6 above)
Comment (mebeim) [reply pages/sunos/snoop.md]: Perhaps we could say this then:

    - Capture packets saving them in a file:

Or even:

    - Save captured packets in a file instead of displaying them:

File: pages/sunos/snoop.md (hunk 7 above)
Comment (mebeim) [reply pages/sunos/snoop.md]: We can go with the American pronunciation :+1: thanks for for the note.
